notebook.ipynb

```diff
@@ -265,63 +266,237 @@
+       "498           529  ...               3                 0                 0   \n",
+       "499           388  ...               2                 0                 0   \n",
        "\n",
-       "   Campaign1Success  Campaign2Success  RecentComplaint  Z_CC  Z_R  \\\n",
-       "0                 0                 0                0     3   11   \n",
-       "1                 0                 0                0     3   11   \n",
-       "2                 0                 0                0     3   11   \n",
-       "3                 0                 0                0     3   11   \n",
-       "4                 0                 0                0     3   11   \n",
+       "     Campaign5Success  Campaign1Success  Campaign2Success  RecentComplaint  \\\n",
+       "0                   0                 0                 0                0   \n",
+       "1                   0                 0                 0                0   \n",
+       "2                   0                 0                 0                0   \n",
+       "3                   0                 0                 0                0   \n",
+       "4                   0                 0                 0                0   \n",
+       "..                ...               ...               ...              ...   \n",
+       "495                 0                 0                 0                0   \n",
+       "496                 0                 0                 0                0   \n",
+       "497                 0                 0                 0                0   \n",
+       "498                 0                 0                 0                0   \n",
+       "499                 0                 0                 0                0   \n",
        "\n",
-       "   LastCampaignResponse  \n",
-       "0                     1  \n",
-       "1                     0  \n",
-       "2                     0  \n",
-       "3                     0  \n",
-       "4                     0  \n",
+       "     Z_CC  Z_R  LastCampaignResponse  \n",
+       "0       3   11                     1  \n",
+       "1       3   11                     0  \n",
+       "2       3   11                     0  \n",
+       "3       3   11                     0  \n",
+       "4       3   11                     0  \n",
+       "..    ...  ...                   ...  \n",
+       "495     3   11                     0  \n",
+       "496     3   11                     0  \n",
+       "497     3   11                     0  \n",
+       "498     3   11                     0  \n",
+       "499     3   11                     0  \n",
        "\n",
-       "[5 rows x 29 columns]"
+       "[500 rows x 29 columns]"
       ]
      },
-     "execution_count": 74,
+     "execution_count": 156,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
     "# Looking at the first five rows\n",
-    "data.head(5)"
+    "data.head(500)"
```

Comment:
Hello, Thanks for the contribution.

This line of code is not following the comment used.